### PR TITLE
complete/task/55/test-content-switcher

### LINKED ISSUE #55: (SP: 1) Write tests for content switcher component
Writing tests don't forget to mock elements if neccessary.

Test cases:

- [ ] it should render with the correct props
- [ ] it should call the onChange function when the switch is clicked
- [ ] it should renders tooltips when tooltip props are passed

### PR DESCRIPTION
**Pull request by PavloChuchman**

### CODE REVIEW

**Review by SeredenkoDS — APPROVED** [review]
Checked it out, the test runs and passes, and I think the code looks alright. Approving.

Oh, and for everyone else, here are the testing commands for your convenience:
git checkout task/55/test-content-switcher-component (on client)
npm test -- contentSwitcher.spec.jsx

**Review by HalynaProlubshchykova — APPROVED** [review]

**Review by grbvtsk — APPROVED** [review]

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Do not create a new component but test the existing one `src/components/app-content-switcher/AppContentSwitcher.tsx`

File: src/components/app-range/AppRange.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import React from 'react'
```
Comment (vlad-khrychov) [inline src/components/app-range/AppRange.tsx:1]: Do we actually need this import?

File: src/components/app-content-switcher/AppContentSwitcher.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { FC } from 'react'
+import React, { FC } from 'react'
```
Comment (vlad-khrychov) [inline src/components/app-content-switcher/AppContentSwitcher.tsx:1]: Do we actually need this import?
```suggestion
import { FC } from 'react'
```

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**